### DISCUSSION TITLE
Command "yarn run build" exited with 137

### DISCUSSION THREAD

**Discussion by hunght** [vercel/community#1584] [Help]
### Summary

I run the nextra blog build on vercel and always got this issue. Seem like it out of memory. I could run on local fine. But can't deploy my app to production cause of this error. Pls help

**Comment by amyegan** [comment]
Hi @hunght! Can you share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)?
